build/linux/debian/dependencies-generator.js

```diff
@@ -10,6 +10,7 @@ const fs_1 = require("fs");
 const os_1 = require("os");
 const path = require("path");
 const dep_lists_1 = require("./dep-lists");
+const manifests = require("../../../cgmanifest.json");
```

Comment:
/cc @rzhao271 fyi

Ensures that these files are in sync with Electron updates.